src/renderer/components/button/button-panel.tsx

```diff
@@ -0,0 +1,13 @@
+import "./button-panel.scss";
+import React from "react";
+
+export function ButtonPannel(props: React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>) {
```

Comment:
If I understand it right, this `ButtonPanel` shows buttons with a `margin-right` added to last button. Is it reasonable to create separate component for such a small purpose? I suggest to add flex styles in place where it's needed without creating `ButtonPanel`

Reply:
The `margin-right` is added to all buttons except that last one.

I was under the impression that such small components were exactly what React's design methodology was about, which is why I created one.

Reply:
For creating gaps (`margin-right`) between buttons you can use following classes on the parent element
```
<div className="flex gaps">...</div>
```
The `.gaps` rule would add neede margins.

Please look if `ButtonPanel` component could be avoided.

Reply:
Tested and removed.